Microsoft.Toolkit/Extensions/ArrayExtensions.cs

```diff
@@ -10,22 +10,21 @@
 namespace Microsoft.Toolkit.Extensions
 {
     /// <summary>
-    /// Helper extension methods for Arrays.
+    /// Helpers for working with arrays.
     /// </summary>
     public static class ArrayExtensions
     {
         /// <summary>
-        /// Fills in values of a multi-dimensional rectangular array to specified value based on the position and size given.
-        /// Ranges given outside the bounds of the array will fill in as much as possible and ignore elements that should appear outside it.
-        /// Won't throw bounds exception, just won't do work if ranges out of bounds.
+        /// Fills elements of a rectangular array at the given position and size to a specific value.
+        /// Positions outside the bounds of the array are ignored.
         /// </summary>
-        /// <typeparam name="T">Type of array values.</typeparam>
-        /// <param name="array">Extended type instance.</param>
+        /// <typeparam name="T">The element type of the array.</typeparam>
+        /// <param name="array">The source array.</param>
         /// <param name="value">Value to fill with.</param>
         /// <param name="row">Row to start on (inclusive, zero-index).</param>
         /// <param name="col">Column to start on (inclusive, zero-index).</param>
-        /// <param name="width">Positive Width of area to fill.</param>
```

Comment:
Why remove the positive from here?

Reply:
felt implicitly positive, but looking again I can see that it's worth having